### ISSUE TITLE
Unable to fetch project runs

### ISSUE THREAD

**Issue by chansonzhang** [TonicAI/tonic_validate#125]
![image](https://github.com/TonicAI/tonic_validate/assets/88643194/489586d2-5ed1-400d-a685-1bf6b8cf1f57)

https://validate.tonic.ai/projects/7b87c225-01b7-40a9-a27f-5ab94ae392cf

**Comment by akamor** [comment]
Hi @chansonzhang . When did this start happening?

**Comment by chansonzhang** [comment]
@akamor Thanks for your help to recover the polluted projects.

I was using `llama-index-evaluation-tonic-validate==0.1.2` which requires `tonic-validate<3.0.0` and the error occurred with `tonic-validate==3.3.0`. I updated to `tonic-validate==3.4.0` and it works now for new project or projects you have recovered.

However, if a project is polluted again with data uploaded with `tonic-validate==3.3.0`, it won't recover anymore even we uploaded with `tonic-validate==3.4.0` again. This is a test project, I can just delete it and use new one, but I'm wondering how did you recover those polluted projects, by manually manipulate some data?
